project/src/demo/puzzle/level/level-rank-bridge-demo.gd

```diff
@@ -0,0 +1,81 @@
+extends Node
+## Calculates the necessary level metadata for the player to attain their expected grade.
```

Comment:
This comment is not accurate.

Reply:
Changed comment, `Calculates the percentage thresholds for different ranks, using the legacy math-based rank model.`